iset.mm

```diff
@@ -72342,6 +72364,156 @@ Similar to the Axiom of Choice (first form) of [Enderton] p. 49.
+      BCRUDTUEUKVJWJVNWMUKVIWIDEAAUKVHWHFAUKUSVTVGWGWOUKVEWEVFWFUNVDBCRURVDBCRU
+      FUDUCTUKVMWLDEAAUKVKWKVLUKUSVTWOSUGTUEUHDEFABUIDEFACUIUJ $.
+  $}
+
+  ${
+    $d A a b c u v $.  $d A a b c x y $.
+    $( Negated equality on a set with decidable equality is a tight apartness.
+       (Contributed by Jim Kingdon, 5-Feb-2025.) $)
+    netap $p |- ( A. x e. A A. y e. A DECID x = y
+        -> { <. u , v >. | ( ( u e. A /\ v e. A ) /\ u =/= v ) } TAp A ) $=
+      ( va vb vc weq wral cv wcel wa wne wbr wn cop df-br wb neeq2 wdc copab wi
+      cxp wss wtap opabssxp a1i neirr neeq1 opelopab2 anidms bitrid mtbiri rgen
+      wo ancoms necom bitrdi bitr4d biimpd rgen2 jca w3a 3adant3 simpr eqnetrrd
+      adantl simplr necomd neqned equequ2 dcbid equequ1 ralbidv simplr1 rspcdva
+      olcd simpll simplr3 exmiddc syl mpjaodan 3adant2 3adant1 orbi12d ad2antlr
+      orcd mpbird ex sylbid ralrimivvva notbid df-ne notbii simpl simprl simprr
+      notnotrdc ralrimivva df-tap syl3anbrc ) ABIZUAZBEJZAEJZDKZELCKZELMXGXHNZM
+      DCUBZEEUDUEZFKZXLXJOZPZFEJZXLGKZXJOZXPXLXJOZUCZGEJFEJZMXQXLHKZXJOZXPYAXJO
+      ZUPZUCZHEJGEJFEJZXQPZFGIZUCZGEJFEJZMEXJUFXKXFXIDCEEUGUHXFXOXTXOXFXNFEXLEL
+      ZXMXLXLNZXLUIXMXLXLQXJLZYKYLXLXLXJRYKYMYLSXIXLXHNZYLDCXLXLEEXGXLXHUJZXHXL
+      XLTUKULUMUNUOUHXTXFXSFGEEYKXPELZMZXQXRYQXQXLXPNZXRXQXLXPQXJLYQYRXLXPXJRXI
+      YNYRDCXLXPEEYOXHXPXLTUKUMZXRXPXLQXJLZYQYRXPXLXJRYQYTXPXLNZYRYPYKYTUUASXIX
+      PXHNZUUADCXPXLEEXGXPXHUJZXHXLXPTUKUQXPXLURUSUMUTVAVBUHVCXFYFYJXFYEFGHEEEX
+      FYKYPYAELZVDZMZXQYRYDUUEXQYRSZXFYKYPUUGUUDYSVEVHUUFYRYDUUFYRMZYDXLYANZXPY
+      ANZUPZUUHFHIZUUKUULPZUUHUULMZUUJUUIUUNYAXPUUNXLYAXPUUHUULVFUUFYRUULVIVGVJ
+      VRUUHUUMMZUUIUUJUUOXLYAUUHUUMVFVKWHUUHUULUAZUULUUMUPUUHFBIZUAZUUPBEYABHIU
+      UQUULBHFVLVMUUHXEUURBEJZAEXLAFIZXDUURBEUUTXCUUQAFBVNVMVOZXFUUEYRVSYKYPUUD
+      XFYRVPVQYKYPUUDXFYRVTVQUULWAWBWCUUEYDUUKSXFYRUUEYBUUIYCUUJYBXLYAQXJLZUUEU
+      UIXLYAXJRYKUUDUVBUUISYPXIYNUUIDCXLYAEEYOXHYAXLTUKWDUMYCXPYAQXJLZUUEUUJXPY
+      AXJRYPUUDUVCUUJSYKXIUUBUUJDCXPYAEEUUCXHYAXPTUKWEUMWFWGWIWJWKWLXFYIFGEEXFY
+      QMZYGYHPZPZYHYQYGUVFSXFYQYGYRPUVFYQXQYRYSWMYRUVEXLXPWNWOUSVHUVDYHUAZUVFYH
+      UCUVDUURUVGBEXPBGIUUQYHBGFVLVMUVDXEUUSAEXLUVAXFYQWPXFYKYPWQVQXFYKYPWRVQYH
+      WSWBWKWTVCFGHEXJXAXB $.
+  $}
+
+  ${
+    $d u v $.  $d x y $.
+    $( Negated equality is a tight apartness on ` 2o ` .  (Contributed by Jim
+       Kingdon, 6-Feb-2025.) $)
+    2onetap $p |- { <. u , v >. |
+        ( ( u e. 2o /\ v e. 2o ) /\ u =/= v ) } TAp 2o $=
+      ( vx vy weq wdc c2o wral cv wcel wa wne copab wtap 2onn elnn mpan2 nndceq
+      com syl2an rgen2 netap ax-mp ) CDEFZDGHCGHGBIZGJAIZGJKUEUFLKBAMNUDCDGGCIZ
+      GJZUGSJZDIZSJZUDUJGJZUHGSJZUIOUGGPQULUMUKOUJGPQUGUJRTUACDABGUBUC $.
+  $}
+
+  ${
+    $d u v $.
+    $( The set ` 2o ` with negated equality as an apartness, has two points
+       apart.  (Contributed by Jim Kingdon, 8-Feb-2025.) $)
```

Comment:
The statement does not mention apartness.  I would be more "agnostic" to say
```
(/) and 1o are two unequal elements of 2o.
```

Reply:
I've accepted that suggestion.